pkg/bitcoin/address.go

```diff
@@ -56,9 +82,9 @@ func (s *Service) EncodeAddress(
 	//
 	// Using addresses encoded from incorrect public keys may lead to
 	// irrevocable fund loss.
-	publicKey, err := btcec.ParsePubKey(request.PublicKey, btcec.S256())
+	loadedPublicKey, err := btcec.ParsePubKey(publicKey, btcec.S256())
 	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, err.Error())
+		return "", err
```

Comment:
Using [`pkg/errors`](https://github.com/pkg/errors) would allow to wrap and give context like:
```
if err != nil {
        return "", errors.Wrapf(err, "Unable to parse public key %s", publicKey)
}
```
Meaningless now, meaningful tomorrow when someone else will try to debug any key encoding related issue.